index.html

```diff
@@ -1,22 +1,237 @@
 	<meta charset="utf-8">
-	<title></title>
+	<title>YouTube Video Player</title>
+	<script src="https://kit.fontawesome.com/11bb635b00.js" crossorigin="anonymous"></script>
+	<style>
+		@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap');
+	</style>
+	<link rel="stylesheet" href="styles.css">
 </head>
+
 <body>
-<div id="nav-bar"></div>
-<div id="video-player">
-	<div id="video-screen"></div>
-	<div id="video-title"></div>
-	<div id="video-description"></div>
-</div>
-<div id="thumbnails">
-	<div id="thumbnail-one"></div>
-	<div id="thumbnail-two"></div>
-	<div id="thumbnail-three"></div>
-	<div id="thumbnail-four"></div>
-	<div id="thumbnail-five"></div>
-</div>
+	<section id="navbar">
+		<div id="hamburger-button">
+			<button class="hamburger">
+				<i class="fas fa-bars"></i>
+			</button>
+			<div id="icon">
+				<button class="yt-icon">
+					<i class="fab fa-youtube" style="color: red;"></i>
+					<p>YouTube</p>
+				</button>
+			</div>
+		</div>
+		<div id="searchbar">
+			<div id="search-box">
+				<input type="text" placeholder="Search">
+			</div>
+			<div id="search-button">
+				<i class="fas fa-search"></i>
+			</div>
+			<div id="google-account">
+				<i class="fas fa-sign-in-alt">Sign In</i>
+			</div>
+		</div>
+	</section>
+	<section id="video-container">
+		<section id="video-space">
+			<section id="video-window">
+				<iframe width="560" height="315" src="https://www.youtube.com/embed/wZDk94l57h8"
+					title="YouTube video player" frameborder="0"
+					allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+					allowfullscreen></iframe>
+				<h2 id="video-title">How I Went from College Professor to Deli Worker</h2>
+				<div id="video-info">
+					<div class="likes" id="views">
+						<p id="number-views">211,665 views</p>
+					</div>
+					<div class="likes">
+						<i class="far fa-thumbs-up"></i>
+						<p id="number-likes">12,568</p>
+					</div>
+					<div class="likes">
+						<i class="far fa-thumbs-down"></i>
+						<p id="number-dislikes">138</p>
+					</div>
+					<div class="likes">
+						<i class="fas fa-share"></i>
+						<p id="share">SHARE</p>
+					</div>
+					<div class="likes">
+						<i class="fas fa-list"></i>
+						<p id="save">SAVE</p>
+					</div>
+					<div class="likes">
+						<i class="fas fa-ellipsis-h"></i>
+					</div>
+				</div>
+				<div id="description-container">
+					<div id="first-part">
+						<div id="channel">
+							<div id="user-image-container">
+								<img id="user-image"
+									src="https://yt3.ggpht.com/ytc/AKedOLT6ck5p3kvuEMSLqzf9aviTcwthTLwoSZnjuJAvhw=s176-c-k-c0x00ffffff-no-rj"
+									alt="This should be a Rick Beato image">
+							</div>
+							<div id="channel-info-container">
+								<p id="user-name">Rick Beato</p>
+								<p id="subscribers">2.59 M subscribers</p>
+							</div>
+						</div>
+						<div id="subscribe-button">
+							<button id="button">SUBSCRIBE</button>
```

Comment:
The `button` should always have a `type` attribute to tell browsers what type of button it is. Please keep this in mind.